libs/mrwebrtc/src/interop/global_factory.cpp

```diff
@@ -182,6 +182,16 @@ rtc::Thread* GlobalFactory::GetWorkerThread() const noexcept {
 #endif  // defined(WINUWP)
 }
 
+rtc::Thread* GlobalFactory::GetSignalingThread() const noexcept {
+  // This only requires init_mutex_ read lock, which must be acquired to access
+  // the singleton instance.
+#if defined(WINUWP)
+  return impl_->signalingThread.get();
```

Comment:
Comment says "Get the WebRTC signaling thread, or NULL if the library is not initialized." but that probably crashes as `impl_` is NULL no?

Reply:
I copied the workerThread one 😄 Will fix according to the docs.

Reply:
Fixed